addons/binding/org.openhab.binding.konnected/README.md

```diff
@@ -0,0 +1,38 @@
+# Konnected Binding
+
+This binding is for interacting with the Konnected Module. A module which intefaces with existing home security sensors. 
+
+https://konnected.io/_
+
+
+## Supported Things
+
+This binding supports one type of thing:  a konnected module.
+
+## Discovery
+
+The binding will auto disocver konnected modules which are attached to the same network as the server running openhab via UPnP.  The binding will then create things for each module discovered which can be added.
+
+## Binding Configuration
+
+There is no configuration required for the binding.
+
+## Thing Configuration
+
+The autodiscovery service of the binding will detect the ipAddress and port of the konnected module.  But once it is added you will need to provide an Authority Token to secure communication between the module and OpenHAB.  
```

Comment:
Please update this README so each sentence starts on it's own line. It shouldn't matter how it is rendered. But that way it is easier to review changes in future pull requests.